cli_reference/admin_cli_operations.adoc

```diff
@@ -0,0 +1,175 @@
+= Administrator CLI Operations
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+This topic provides information on the administrator CLI operations and their syntax. You must link:get_started_cli.html[setup and login] with the CLI before you can perform these operations.
+
+The administrator CLI uses the `oadm` command, and is used for administrator operations. This differs from the link:basic_cli_operations.html[developer CLI], which uses the `oc` command, and is used for basic, project-level operations. 
+
+[[common-operations]]
+
+== Common Operations
+The administrator CLI allows interaction with the various objects that are managed by OpenShift. Many common `oadm` operations are invoked using the following syntax:
+
+----
+$ oadm <action> <option>
+----
+
+This specifies:
+
+- An `<action>` to perform, such as `new-project` or `router`.
+- An available `<option>` to perform the action on as well as a value for the option. Options include `--output` or `--credentials`.
+
```

Comment:
Same comment here now in the Admin CLI topic, about considering moving "Object Types" from the very bottom of the topic to here, between "Common Operations" (which sets up the most basic form of how the commands are structured) and "Basic CLI Operations" (which begins showing commands that make use of the object types).